distribution/docs/src/main/resources/content/_appReferences/mg-geowebcache.adoc

```diff
@@ -137,12 +135,12 @@ Add a new Imagery Provider in the ${admin-console}:
 . Select *Standard Search UI* configuration.
 . Click the *Add* button next to *Imagery Providers*
 . Enter configuration for Imagery Provider in new textbox:
-. `{"type" "WMS" "url" "${secure_url}/geowebcache/service/wms" "layers" ["states"] "parameters" {"FORMAT" "image/png"} "alpha" 0.5}`
+. `{"type": "WMS", "order": 0, "url": "${secure_url}/geowebcache/service/wms", "layers": ["states"], "parameters": {"FORMAT": "image/png", "VERSION": "1.1.1"}, "alpha" 0.5}`
```

Comment:
Same question about the Standard Search UI format.  If they are different we may want to provide both configuration styles in this documentation.

Reply:
reverting